Signal/src/call/CallAudioService.swift

```diff
@@ -54,8 +54,14 @@ import Foundation
 
     private func ensureIsEnabled(call: SignalCall) {
         // Auto-enable speakerphone when local video is enabled.
-        if call.isSpeakerphoneEnabled || call.hasLocalVideo {
-            setAudioSession(category: AVAudioSessionCategoryPlayAndRecord, options: .defaultToSpeaker)
+        if call.hasLocalVideo {
+            setAudioSession(category: AVAudioSessionCategoryPlayAndRecord,
+                            mode: AVAudioSessionModeVideoChat,
```

Comment:
The docs are very vague about the effects of AVAudioSessionModeVideoChat and AVAudioSessionModeVoiceChat. It's not just which mics that are affected but also the audio processing.